### DISCUSSION TITLE
Vitest coverage with husky

### DISCUSSION THREAD

**Discussion by brunoandradebr** [vitest-dev/vitest#3859] [Q&A] [answered]
Is that possible to set vitest coverage threshold to block commit if it's lower to settings using husky/lint-staged ?

**Comment by AriPerkkio** [comment]
Sure. Husky should prevent the commit when given command fails. Vitest will exit with error code when coverage thresholds are not met.

**Reply by brunoandradebr** [reply]
But is it necessary to explicity define coverage settings in vite config file ? cause mine is passing even not covering 80% (default)

**Reply by AriPerkkio** [reply, marked as answer]
Threshold options are disabled by default. You can also pass them as CLI arguments, e.g. `vitest run --coverage.enabled --coverage.lines=80`.
